code/game/turfs/open/lava.dm

```diff
@@ -151,7 +151,7 @@
 
 /turf/open/lava/proc/can_burn_stuff(atom/movable/burn_target)
 	if(burn_target.movement_type & (FLYING|FLOATING)) //you're flying over it.
-		return isliving(burn_target) ? LAVA_BE_PROCESSING : LAVA_BE_IGNORING
+		return isliving(burn_target) ? LAVA_BE_IGNORING : LAVA_BE_PROCESSING 
```

Comment:
this makes it so objects that fly can burn is this intended

Reply:
jesus christ, the pre refactor code had it as anything with FLYING or FLOATING wasn't burnt, i should probably just remove the whole statement